app/src/main/java/org/mozilla/firefox/vpn/MockedGuardianComponent.kt

```diff
@@ -18,10 +19,12 @@ class MockedGuardianComponent(
 
     private val sessionManager = SessionManager(prefs)
 
+    private val referralManager = ReferralManager(coreComponent.app.applicationContext, prefs)
+
```

Comment:
only need to use `app.applicationContext` ?

Reply:
the application context is enough
see: https://developer.android.com/reference/com/android/installreferrer/api/InstallReferrerClient#newBuilder(android.content.Context)